src/components/databaseSelect.js

```diff
@@ -22,3 +38,11 @@ export default class DatabaseSelect extends Component {
     )
   }
 }
+
+const mapState = state => ({
+  dbList: state.get('dbList')
+})
+
+const mapDispatch = {fetchDbNames}
```

Comment:
Is it possible to write out the logic for mapDispatch. I don't easily follow the short form you used here, and will probably have to ask more questions than they're worth to sort it out whenever it comes up.

Reply:
Hey so for mapDispatch, it’s exactly the same as usual - the short notation just automatically passes the thunk to dispatch. So instead of writing 
const mapDispatch = (dispatch) => {
 return {
    fetchDbNames: () => dispatch(fetchDbNames)
  }

you can just write mapDispatch = {fetchDbNames}
and it automatically passes it to dispatch
so it does the exact same thing because the name of our thunk (fetchDbNames) is exactly the same as the prop we are putting on the object

so now we have access to it as this.props.fetchDbNames

It’s a shorthand that saves a lot of code and Omri also recommended so I’d like to use it if possible